lib/ui/pocket_paint.dart

```diff
@@ -10,51 +13,20 @@ class PocketPaint extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: Text(title), centerTitle: false),
-      body: const DrawingBoard(),
-      bottomNavigationBar: const _BottomNavigationBar(),
-    );
-  }
-}
-
-class _BottomNavigationBar extends StatelessWidget {
-  const _BottomNavigationBar({Key? key}) : super(key: key);
-
-  @override
-  Widget build(BuildContext context) {
-    return BottomNavigationBar(
-      type: BottomNavigationBarType.fixed,
-      currentIndex: 1,
-      items: [
-        BottomNavigationBarItem(
-          label: "Tools",
-          icon: IconButton(
-            onPressed: () {},
-            icon: const Icon(Icons.handyman),
-          ),
+      appBar: AppBar(
+        title: Text(
+          title,
+          style: const TextStyle(
+              color: Colors.white, fontWeight: FontWeight.w500, fontSize: 20),
```

Comment:
I think this can also be moved to the parent theme. Also, please add trailing commas and reformat the file when the args exceed threshold line width.
```suggestion
          style: const TextStyle(
              color: Colors.white,
              fontWeight: FontWeight.w500, 
              fontSize: 20,
          ),
```